src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.SafeguardingModel
+@{
+}
+
+<style>
+    a.nhs:link {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:visited {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:hover {
+        color: white;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    a.nhs:active {
+        color: black;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    .interrupt-panel {
+        width: 100%;
+        padding: 30px;
+        background: #005ea5;
+        color: #fff !important;
+        overflow: hidden;
+        width: auto;
+    }
+
+        .interrupt-panel h1, .interrupt-panel p, .interrupt-panel h2, .interrupt-panel ul {
+            color: #fff !important;
+        }
+
+</style>
+
+@section Back {
+    <a asp-page="/ProfessionalReferral/LocalOfferDetail" asp-route-id="@Guid.NewGuid().ToString()" asp-route-serviceId="@Model.Id" asp-route-name="@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="interrupt-panel">
+            <h1 class="govuk-heading-l">
+                Do not use this service to report safeguarding concerns
+            </h1>
+            <p>Use the <a href="https://nhssafeguarding.app/?nocache=0.23461449202295054" id="nhs-app" class="nhs">NHS safeguarding app</a> for guidance on reporting safeguarding concerns.</p>
+        </div>
+
+        <form method="post" novalidate data-validate='my-form-validation'>
```

Comment:
Is this form needed? (For the next PR perhaps?) 

At the moment, it looks like it's not needed. (The button isn't a submit button, there's no form data and no OnPost method.)

What's it validating? Also, I can't see anything related to `my-form-validation`?

Reply:
Yep Consent Form is next